django/db/models/fields/__init__.py

```diff
@@ -1812,14 +1812,22 @@ def _check_max_length_warning(self):
     def validators(self):
         # These validators can't be added at field initialization time since
         # they're based on values retrieved from `connection`.
-        range_validators = []
+        _validators = super(IntegerField, self).validators
```

Comment:
I think a trailing underscore rather than a leading one might be a more common convention in Django for this case.